docs/UserGuide.md

```diff
@@ -304,6 +437,42 @@ sort payment
 Sort complete!
 ```
 
+### - Display instructions on how to use Classify `help`
+Prints instructions which detail how to use all the features of Classify.
+
+Format: `help`
+
+```
+Example:
+help
+add                         Adds a student to the student list, expects a name, grade and lessons attended, can be used directly with a name e.g. add [name]
+edit                        Edits a students details, expects a name, can be used directly with a name e.g. edit [name]
+view                        Views a students details, expects a name, can be used directly with a name e.g. add [name]
+delete                      Deletes a student from the student list, expects a name, can be used directly with a name e.g. add [name]
+restore                     Restore a student deleted within the current session, expects a name, can be used directly by restore [name].
+undo                        Restores the last student deleted in the current session.
+list                        Displays the list of all students
+                            Currently available types: Whole student list, with total classes attended or by certain subject only
+bye                         Exits Classify
+sort                        Sorts the student list by the input parameter, expects an attribute to sort by, can be used directly by sort [type].
+                            Currently available types: name, classes, payment
+archive                     Removes the specified student from the list and archives them, can be used directly by archive [name].
+unarchive                   Removes the specified student from the archive and adds them to the list, can be used directly by unarchive [name].
+help                        Prints this help message
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
+
+### - Exit Classify `bye`
+Exits the Classify program.
+
+Format: `bye`
+```
+Example:
+bye
+Hope you've had a productive day. See you again! Goodbye!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
+
```

Comment:
Great addition, will need to modify the list function to include the 3 other types